docs/pages/components/Notice.mdx

```diff
@@ -0,0 +1,192 @@
+---
+title: Notice
+description: A notification banner - similar to a Toast, but allows for more flexibile positioning and control.
+source: https://github.com/dequelabs/cauldron/blob/develop/packages/react/src/components/Notice/Notice.tsx
+---
+
+import { Notice, Icon } from '@deque/cauldron-react';
+
+```js
+import { Notice } from '@deque/cauldron-react';
+```
+
+## Examples
+
+### Notice Component
+
+The `Notice` component is used to display a notification banner. It is similar to the [Toast](./Toast) component,
+but can be used in more flexible ways.
+
+For example, the `Notice` component can be used to display a notification banner within a [Panel](./Panel) component
+or anywhere else in the DOM without needing to be absolutely/statically positioned.
+
+In addition, the `Notice` component has only limited dynamic functionality built in, allowing developers to
+have full control over the underlying element via its `forwardRef`, while still having useful base styling..
+
+```jsx example
+<Notice type="info" title='Information'>
```

Comment:
Small nit: We typically have the variants listed out separately. Having each variant broken out allows for a copying of a single variant and breaks out each variant so it can appear/get linked via the table of contents.